src/AddTodoForm.jsx

```diff
@@ -1,10 +1,16 @@
 import React from "react";
 
-const AddTodoForm = () => {
+const AddTodoForm = ({ onAddTodo }) => {
+  const handleAddTodo = (event) => {
```

Comment:
This handler is missing the callback function. You need to invoke onAddToDo and pass todoTitle, otherwise the newTodo state in App.jsx won't ever update